doc/api.md

```diff
@@ -2,31 +2,89 @@
 
 ***If you are running the server and want to enable the API, set the environment variable `API_ENABLED=true`.***
 
-This API is currently in a very limited alpha, and is not enabled in any of our deployments. Note that this API may change at any time, without warning.
+The API design is evolving. Backwards-incompatible changes will result in a version change.
+
+The lastest version is *v2*.
+
+## Clients
```

Comment:
maybe include this in the `Authentication` section to clarify that it's an alternative to auth-ing manually